pkgs/shells/fish/plugins/nvm.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+}:
+
+buildFishPlugin rec {
+  pname = "nvm";
+  version = "2.2.16";
+
+  src = fetchFromGitHub {
+    owner = "jorgebucaran";
+    repo = "nvm.fish";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-B7Ze0a5Zp+5JVsQUOv97mKHh5wiv3ejsDhJMrK7YOx4=";
+  };
+
+  meta = {
+    description = "Node.js version manager you'll adore, crafted just for Fish";
```

In Nixpkgs we like to be boring and impersonal
```suggestion
    description = "Node.js version manager crafted just for Fish";
```